hiero-ledger/hiero-docs
# Remove Duplicate Files whose only difference is naming in lowercase/uppercase

### The problem
The problem is there are files present in .gitbook/assets that are duplicated file names that are only different in upper case/ lower case. This causes problems on case-insensitive file systems (MacOS) which cannot tell the difference between the two files. This confuses `git` and shows there are changes even though no changes are made. This may prevent people from being able to make commits and push because changes will always be detected. 

### How to fix
You can use the following command in the `.gitbook/assets` directory to do a case-insensitive search of duplicate file-names.

`find . -maxdepth 1 | sort -f | uniq -di  `

The only two files that were found were the following

![image](https://github.com/hedera-dev/preview-hedera-docs/assets/18725916/dc42271d-5450-40c7-8888-9bfa7b2a52d6)

### What was done in this PR
Remove these duplicate files and ensured the ones being used in the docs pages are renamed or still exist to prevent broken images. None of the removed files were in use. This fixes case-insensitive repository clones from detecting changes for files that don't exist. 

### How to test this PR
Go to the /README.md page to make sure that the smart contracts.png image is not broken
Go to networks/mainnet/mainnet-nodes/node-requirements/README.md and make sure the Network-topology.jpg diagram is still there.

Note: Whether we keep the lower case or upper case does not matter. The goal is to ensure we don't have two files where the only difference is the casing. This may still be broken on clones and if anyone is still having issues they may have to delete their clone and re-clone the repository.
